compiler/src/dmd/file_manager.d

```diff
@@ -48,50 +120,59 @@ nothrow:
     *      the found file name or
     *      `null` if it is not different from filename.
     */
-    static const(char)[] lookForSourceFile(const char[] filename, const char*[] path)
+    const(char)[] lookForSourceFile(const char[] filename, const char*[] path)
     {
         //printf("lookForSourceFile(`%.*s`)\n", cast(int)filename.length, filename.ptr);
         /* Search along path[] for .di file, then .d file, then .i file, then .c file.
         */
+        // see if we should check for the module locally.
+        bool checkLocal = packageExists(filename);
```

Comment:
```suggestion
        bool checkingLocal = packageExists(filename);
```
?

Reply:
Huh? why is this name change important?

Reply:
It's not important just made all those if statements slightly nicer to my eyes

Reply:
OK, I disagree, it doesn't read any nicer for me.